libs/components/indicators/src/lib/modules/wait/wait.service.ts

```diff
@@ -162,14 +163,17 @@ export class SkyWaitService {
   }
 
   #clearPageWait(isBlocking: boolean): void {
-    if (waitComponent) {
-      if (isBlocking) {
-        pageWaitBlockingCount = 0;
-        waitComponent.hasBlockingWait = false;
-      } else {
-        pageWaitNonBlockingCount = 0;
-        waitComponent.hasNonBlockingWait = false;
+    // Wait for the component to be created before clearing it.
+    this.#windowSvc.nativeWindow.setTimeout(() => {
```

Comment:
I do wonder if we will have consumers with failing tests after this change due to the addition of async code here that may cause `timers still in queue` errors for `fakeAsync` tests. Will be a good test for the update runner.

Reply:
I'm honestly surprised it hasn't broken our own tests until now, since every other method in the wait service is wrapped in a `setTimeout`.